airflow/settings.py

```diff
@@ -722,6 +743,16 @@ def import_local_settings():
         log.info("Loaded airflow_local_settings from %s .", airflow_local_settings.__file__)
 
 
+def mask_conf_values():
+    from airflow.utils.log.secrets_masker import mask_secret
+
+    for env in ENV_VARIABLES_TO_MASK:
+        section, key = conf.get_section_and_key_for_env(env)
```

Comment:
The reason this is done is so that if the config is set in any other way than env, we can detect it and mask it